account_central_billing/views/res_partner.xml

```diff
@@ -141,10 +141,10 @@
                                         <field name="comment" placeholder="internal note..."/>
                                     </group>
                                 </group>
-                                <field name="supplier" invisible="True"/>
-                                <field name="customer" invisible="True"/>
+                                <field name="supplier_rank" invisible="True"/>
+                                <field name="customer_rank" invisible="True"/>
                                 <field name="lang" invisible="True"/>
-                                <field name="image" invisible="True"/>
+                                <field name="image_medium" invisible="True"/>
```

Comment:
Are you sure this is right? It seems strange to replace a real field with a deprecated functional one. I'd have expected it to be the main image field (image_1920 from memory), if it is needed at all. In any case why are we downloading and hiding an invisible field? Anyway, just review this kanban, it basically is meant to be a simple mirror of the equivelant kanban view for regular child contacts I think.